lte/gateway/Makefile

```diff
@@ -95,19 +82,6 @@ restart: stop start ## Restart all services
 
 run: build restart ## Build and run all services
 
-status: ## Status of all services
```

Comment:
Should these (status, log, logs, help) be removed from the the `.PHONY` line at the top of the file too?

Reply:
Done.